pipeline-modules/app-service/gcp/custom-step.yaml

```diff
@@ -0,0 +1,62 @@
+provisioner: gcp
+name: custom-step
+version: 1
+revision: 1
+displayName: Custom Step
+description: This plugin is to run any script or command within a personalized/custom/Any docker image.
+target: ""
+category: common
+keywords:
+  - custom
+  - docker
+  - bash
+
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    step_id: 
+      title: Cloud build Step name
+      description: Define your Cloud build step name
```

Comment:
I would remove the reference to cloud-build. The less the user know about the implementation the better. Also does it bring any benefit for the user to set this? Can they refer to this id? Maybe for documentation - I would probably call it step_name. Otherwise we could just generate a unique one for them.

Reply:
updated